packages/headless/src/api/analytics/commerce-analytics.ts

```diff
@@ -6,21 +6,21 @@ import {
 import {SearchEventRequest} from 'coveo.analytics/dist/definitions/events';
```

Comment:
In fact I think we could delete this file altogether!

Reply:
Done in https://github.com/coveo/ui-kit/pull/4087/commits/515aa44654a616177653b81947a224d210dd263a